src/classic.js

```diff
@@ -36,8 +32,7 @@ export default class ClassicEditor extends StandardEditor {
 
 		this.data.processor = new HtmlDataProcessor();
 
-		this.ui = new ClassicEditorUI( this );
-		this.ui.view = new BoxedEditorUIView( this.locale );
+		this.ui = new ClassicEditorUIView( this, this.locale );
```

Comment:
>I've got mixed feelings. We've got editor.editing.view but you propose editor.view 

This is one more argument why we should rename editing view (see https://github.com/ckeditor/ckeditor5-engine/issues/622). Calling the property `ui` will help, but not much. It sill will be a view which use (editing) view as a model.